sections/adguard-specific/annoyances.txt

```diff
@@ -46,6 +46,7 @@ myonlineradio.hu#$#.js-appAdv { display: none !important; }
 myonlineradio.hu#$#.cover-slider-cont { display: block !important; }
 noizz.hu#$#.leavingPopupLayer { display: none !important; }
 noizz.hu#$#body { overflow: auto !important; }
+unideb.hu#?#style:contains(eu-cookie-withdraw-banner)
```

Comment:
You mean the following rule?

```adblock
unideb.hu$$style:contains(eu-cookie-withdraw-banner)
```

You can also combine with [this rule](https://github.com/hufilter/hufilter/pull/585#pullrequestreview-2816076490)